internal/nodes/dequeue_ingestion.go

```diff
@@ -291,11 +291,30 @@ func handleRoute(key string, fullsync bool, routeNames []string) {
 	sharedQueue := utils.SharedWorkQueue().GetQueueByName(utils.GraphLayer)
 	utils.AviLog.Infof("key: %s, msg: route found: %v", key, routeNames)
 	for _, route := range routeNames {
-		nsroute, nameroute := getIngressNSNameForIngestion(objType, namespace, route)
-		utils.AviLog.Infof("key: %s, msg: processing route: %s", key, route)
-		HostNameShardAndPublish(utils.OshiftRoute, nameroute, nsroute, key, fullsync, sharedQueue)
+		if strings.Contains(route, lib.ShardVSSubstring) {
```

Comment:
Would route names have a a match with Shard VS name ? Also in this case why are we skipping HostNameShardAndPublish ?